sdk/core/azure-core/src/test/java/com/azure/android/core/credential/AccessTokenTest.java

```diff
@@ -0,0 +1,39 @@
+package com.azure.android.core.credential;
+
+import org.junit.Test;
+import org.threeten.bp.OffsetDateTime;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class AccessTokenTest {
+    private String token = "testToken";
+    private OffsetDateTime time; // Expiration is set to 2 minutes before the given time.
```

Comment:
This is no longer true, does this test file need to be updated?

Reply:
I thought I had pushed the change, let me do that right now.